app/src/main/java/io/github/polysmee/appointments/fragments/AppointmentCreationAddUserFragment.java

```diff
@@ -129,9 +150,18 @@ public void reset() {
      *                  it from the removed participant list
      */
     private void inviteButtonBehavior(View view) {
-        //For now we only get the input from the SearchView without checking it as the objective wasn't to add the database component, this will be done later
         String s = searchInvite.getText().toString();
-        if(!invites.contains(s) && !s.isEmpty()) {
+        if(!users.contains(s)) {
+            builder.setMessage("User not found")
+                    .setCancelable(false)
+                    .setPositiveButton("Ok", null);
+
+            AlertDialog alert = builder.create();
+            alert.setTitle("Error");
+            alert.show();
+        }
+
+        else if(!invites.contains(s) && !s.isEmpty()) {
             invites.add(s);
```

Comment:
I think the condition on line 164 !s.isEmpty() will never be true as if s is empty the condition in line 154 will be true. 
The only way 154 will not be true before is If a user is called "" which result in the call empty to it to true then the condition !s.isEmpty() would be true but you will need to consider "" as a real username and not as nothing added